app/blocks/external/serviceScript.js

```diff
@@ -47,7 +48,7 @@ class ServiceScript extends ExternalBlock {
     return this._ensurePromise(this.script(this.options)).then(() => {
       this.start()
     }).catch((error) => {
-      this.logger.log('error', 'Script failed', { error })
+      this.logger.log('error', 'Script failed', { error: util.inspect({error}) })
```

Comment:
Maybe it would be better to convert this to `this.logger.error('Script failed', error)` then add the `util.inspect` to `app/lib/logger.js` ?